Client/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+	id 'fabric-loom' version '1.2-SNAPSHOT'
+}
+
+sourceCompatibility = targetCompatibility = 17
```

Comment:
this and other parts of the build.gradle are deprecated. See lotas common.gradle for info (mainly base block)